frontend/src/modules/Datasets/index.js

```diff
@@ -0,0 +1,5 @@
+export const DatasetsModule = {
+  moduleDefinition: true,
+  name: 'datasets',
+  isEnvironmentModule: false
```

Comment:
Why dashboard is environment module, but datasets not? :)

Reply:
an environment module is a module that can be toggled in the environment view. So far the only environment modules that users can enabled/disable in their environments are: dashboards, datapipelines, MLStudio, Notebooks